### PR TITLE
[WIP] Added test for product selection when explicitly set in the profile

### PR DESCRIPTION
**Pull request by mchf**
SLE-15 only test

currently not working - either fix regcode for SLED-15 or re-write and use know one for SLES-15. I'd prefer the first option bcs at least at some point SLES was expected by default by some components.

### CODE REVIEW

File: aytests/check_product.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e -x
+
+[[ rpm -qf /etc/os-release =~ sled.* ]]
```
Comment (jreidinger) [inline aytests/check_product.sh]: why not call file `check_sled` ? It does not check if product is set, but it is sled.

Comment (mchf) [reply aytests/check_product.sh]: bcs I didn't know about such script

Comment (jreidinger) [reply aytests/check_product.sh]: @mchf I am not talking about what is inside file, but why test is called `check_product.sh` and not `check_sled_product.sh`?

Comment (mchf) [reply aytests/check_product.sh]: is there something similar for all products?

Comment (mchf) [reply aytests/check_product.sh]: agh, didn't get it ... I'm already in passive == meeting mode

i can rename it, don't care much

**Comment by imobachgs** [comment]
In general, it looks good. We could extend aytests to read values from environment variables so you can set the code without making it public. I does not solve the issue you are having though. What do you think?

Please, solve the conflict.

**Comment by teclator** [comment]
@mchf is there some replacements to these tests? if it is still valid and needed then please solve the conflicts.

File: aytests/sle15_product_selection_1.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+      <second_stage config:type="boolean">false</second_stage>
+    </mode>
+  </general>
+
+  <software>
+    <products config:type="list">
+      <product>SLES15</product>
```
Comment (teclator) [inline aytests/sle15_product_selection_1.xml:19]: It should be just SLES

**Comment by teclator** [comment]
@mchf the conflicts need to be fixed and maybe the profiles need also some adaptations, closed it by not. Reopen it if you plan to fix.

**Pull request closed without merging**